library/classes/RXList.class.php

```diff
@@ -67,38 +67,37 @@ function getPage($query)
         $response = oeHttp::get($url, ['name' => $query]);
         $buffer = $response->body();
         return $buffer ? $buffer : false;
-    } // end function RxList::getPage
+    }
 
     function get_list($query)
     {
         $page = RxList::getPage($query);
         $tokens = RxList::parse2tokens($page);
         $hash = RxList::tokens2hash($tokens);
```

Comment:
Since the current class methods don't declare access modifiers these calls are confusing and inefficient.
Should be `$this->getPage($query);` and so forth.

While we're here, may as well fix this class structure. i.e
`public function get_list($query)` same for other methods.

Or leave and I'll refactor when I add the RxCUI drug lookup if codes are installed.
Orrr, do ya just want me to leave you alone!:)